### PR TITLE
Fix clipping in android of dismiss icon for teams

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 
Another issue where ios allows overflow and android doesn't

### CODE REVIEW

File: shared/teams/main/banner.js [hunk 1]
```diff
@@ -57,12 +57,28 @@ const Banner = ({onReadMore, onHideBanner}: Props) => (
...
       </Text>
     </Box>
-    <Icon
-      type="iconfont-close"
-      onClick={onHideBanner}
-      style={{position: 'absolute', right: globalMargins.tiny, top: globalMargins.tiny}}
-    />
+    <Box
+      style={{
+        ...closeIconStyle,
```
Comment (chrisnojima) [inline shared/teams/main/banner.js]: might as well put the position / background into the style instead of this spread all the time

Comment (MarcoPolo) [reply shared/teams/main/banner.js]: also whoops didn't mean to commit the bg color!!!

Comment (chrisnojima) [reply shared/teams/main/banner.js]: might as well not spread it then :)

File: shared/teams/main/banner.js [hunk 2]
```diff
@@ -57,12 +57,28 @@ const Banner = ({onReadMore, onHideBanner}: Props) => (
...
+      }}
+    >
+      <Icon type="iconfont-close" onClick={onHideBanner} />
+    </Box>
   </Box>
 )
 
+let closeIconStyle = isMobile
```
Comment (chrisnojima) [inline shared/teams/main/banner.js]: likely better to do
{
position: absolute,
...(isMobile ? {} : {})
}

**Pull request merged**